contributors/design-proposals/api-proposal-design-template.md

```diff
@@ -0,0 +1,84 @@
+# Description of your Api
+
+## Owners
+
+Sig owner: list of sigs that will own the design, implementation, and support
+Individual owners: list of individual GitHub users that will own the design, implementation, and support
+
+## Abstract
+
+4-8 sentence description of the problem and solution.  Bullet points welcome.
```

Comment:
It would also be useful to sketch a more general proposal template so that we can evaluate the similarities / differences.